app/database/admin/data_operator/test.ts

```diff
@@ -0,0 +1,317 @@
+                dbType: DatabaseType.SERVER,
+                serverUrl: 'https://appv2.mattermost.com',
+            },
+        });
+        expect(db).toBeTruthy();
+
+        const preparedRecords = await operateCustomEmojiRecord({
+            db: db!,
+            optType: OperationType.CREATE,
+            tableName: CUSTOM_EMOJI,
+            value: {id: 'emo-1', name: 'emoji'},
+        });
+
+        expect(preparedRecords).toBeTruthy();
+        expect(preparedRecords!.collection.modelClass.name).toMatch('CustomEmoji');
+    });
+
+    it('=> should return an array of type Role for operateRoleRecord', async () => {
+        expect.assertions(3);
+
+        const db = await DatabaseManager.createDatabaseConnection({
+            shouldAddToDefaultDatabase: true,
+            databaseConnection: {
+                actionsEnabled: true,
+                dbName: 'community mattermost',
+                dbType: DatabaseType.SERVER,
+                serverUrl: 'https://appv2.mattermost.com',
+            },
+        });
+        expect(db).toBeTruthy();
+
+        const preparedRecords = await operateRoleRecord({
+            db: db!,
+            optType: OperationType.CREATE,
+            tableName: ROLE,
+            value: {id: 'role-1', name: 'role-name-1', permissions: []},
+        });
+
+        expect(preparedRecords).toBeTruthy();
+        expect(preparedRecords!.collection.modelClass.name).toMatch('Role');
+    });
+
+    it('=> should return an array of type System for operateSystemRecord', async () => {
+        expect.assertions(3);
+
+        const db = await DatabaseManager.createDatabaseConnection({
+            shouldAddToDefaultDatabase: true,
+            databaseConnection: {
+                actionsEnabled: true,
+                dbName: 'community mattermost',
+                dbType: DatabaseType.SERVER,
+                serverUrl: 'https://appv2.mattermost.com',
+            },
+        });
+        expect(db).toBeTruthy();
+
+        const preparedRecords = await operateSystemRecord({
+            db: db!,
+            optType: OperationType.CREATE,
+            tableName: SYSTEM,
+            value: {id: 'system-1', name: 'system-name-1', value: 'system'},
+        });
+
+        expect(preparedRecords).toBeTruthy();
+        expect(preparedRecords!.collection.modelClass.name).toMatch('System');
+    });
+
+    it('=> should return an array of type TermsOfService for operateTermsOfServiceRecord', async () => {
+        expect.assertions(3);
+
+        const db = await DatabaseManager.createDatabaseConnection({
+            shouldAddToDefaultDatabase: true,
+            databaseConnection: {
+                actionsEnabled: true,
+                dbName: 'community mattermost',
+                dbType: DatabaseType.SERVER,
+                serverUrl: 'https://appv2.mattermost.com',
+            },
+        });
+        expect(db).toBeTruthy();
+
+        const preparedRecords = await operateSystemRecord({
```

Comment:
```suggestion
        const preparedRecords = await operateTermsOfServiceRecord({
```

Reply:
This one does not seem to enforce correct table name

Reply:
Thanks for picking up this one.  I have you a meeting invite for tomorrow.  I would like us to walk through the code and I will show you why it didn't fail.

Reply:
@avinashlng1080 I think you sent the invite for March 9th

Reply:
Hehe - I don't why the calendar was there.  I updated the invite :) Thanks Joseph